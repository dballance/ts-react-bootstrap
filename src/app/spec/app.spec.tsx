import { shallow } from 'enzyme';
import React from 'react';

import App from '../app';

describe('App ', () => {
  it('should render without throwing', () => {
    const wrapper = shallow(<App />);

    expect(wrapper.find('div').length).toBe(1);
  });

  it('should render text', () => {
    const wrapper = shallow(<App/>);

    expect(wrapper.find('div').text()).toBe('Hello World!');
  });

  it('should match the snapshot', () => {
    const wrapper = shallow(<App/>);

    expect(wrapper).toMatchSnapshot();
  });
});
