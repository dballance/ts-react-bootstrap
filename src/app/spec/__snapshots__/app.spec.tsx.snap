// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App  should match the snapshot 1`] = `
<div
  css={
    Object {
      "map": undefined,
      "name": "1kompxv",
      "next": undefined,
      "styles": "
  color: white;
  background-color: blue;
",
    }
  }
>
  Hello World!
</div>
`;
